portfolio/src/main/webapp/index.html

```diff
@@ -6,52 +6,71 @@
         <link rel="stylesheet" href="style.css">
         <script src="script.js"></script>
     </head>
-    <body onload="getComments()">
+    <body onload="getLogin()">
+        <div id="nav">
+            <h1>♥DOLAPO'S  PORTFOLIO</h1>
+            <ul id="nav-list">
+                <a id="current" href="#"><li>HOME</li></a>
+                <a href="./pages/gallery.html"><li>GALLERY</li></a>
+                <a href="./pages/resume.html"><li>RESUME</li></a>
+                <a id="login" href="#"><li>LOGIN</li></a>
+            </ul></b>
+        </div>
         <div id="content">
-            <h1>My Portfolio</h1>
-            <h2>About Me</h2>
-            <img id="me" src="./images/bitmoji.png">
-            <p>
-                My name is Oluwadolapo "Dolapo" Atobiloye. I am a second-year 
-                Bachelor's of Science student studying Computer Science at 
-                Rochester Insitute of Technology with a minor in Game Design 
-                & Development and an Immersion in Theater Arts.
-            </p>
-            <p>
-                I have experience as a Software Engineering intern in Terragon 
-                Group located in Lagos Nigeria. I have also completed multiple 
-                computer science and web design courses in my time at RIT.
-            </p>
-            <p>
-                I am a huge fan of musical theater and while I'm not much of a 
-                singer, I consider myself a budding actress. I originated the 
-                role of Yashira in the origianl play AI-Pollo by David Munnell.
-                I also performed as #8 in RIT performing arts scholarship's 
-                production of "The Wolves". I try to abide by the motto "Live
-                while you can", which basically means try everything and is how 
-                I got into musicals in the first place. Check out my 
-                <a href="./pages/gallery.html">gallery.</a>
-            </p>
-            <p>
-                You can find me on LinkedIn and GitHub through the links below:
-            </p>
-            <button id="lin" onclick="window.location.href='https://www.linkedin.com/in/dolapo-atobiloye'"></button>
-            <button id="ghub" onclick="window.location.href='https://www.github.com/dxa6589'"></button>
+            <div id="comments" class="infoBox">
+                <h2>Comments</h2>
+                <p id="comments-prompt"></p>
+                <form id="comments-form" action="/data" method="POST">
+                    <!--label for="name">Name:</label>
+                    <input type="text" id="name" name="name"><br-->
```

Comment:
Any particular reason you have these commented out lines in the html? If the code isn't doing work, its best to either remove it or explain why its been left in.

Reply:
Oh right. That is for nickname functionality which I was not able to fully implement. I'll make sure to explain commented off blocks in future pull requests.